pkg/util/log/log.go

```diff
@@ -1,57 +1,123 @@
 package log
 
 import (
+	"fmt"
+
+	"github.com/elastic/beats/v7/libbeat/logp"
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	debugSelector = "apic-agents"
+	traceSelector = "apic-agents-trace"
+)
+
+var isLogP bool
```

Comment:
Could have maybe set a default value of either T or F. Then only need to call fn to change value if so desired.

Reply:
golang defaults all bools to false, that is why I only wanted the set function

Reply:
oh, right.